jvm/src/main/scala/com/avast/sst/jvm/execution/ExecutorModule.scala

```diff
@@ -53,56 +53,71 @@ class ExecutorModule[F[_]: Sync](
 
 object ExecutorModule {
 
+  private final val DefaultBlockingExecutorConfig = ThreadPoolExecutorConfig(0, Int.MaxValue, allowCoreThreadTimeout = true)
+
   /** Makes [[com.avast.sst.jvm.execution.ExecutorModule]] with default callback executor and extra [[cats.effect.Blocker]] executor
     * for blocking operations.
     */
   def makeDefault[F[_]: Sync]: Resource[F, ExecutorModule[F]] = {
     for {
       numOfCpus <- Resource.eval(Sync[F].delay(Runtime.getRuntime.availableProcessors))
       coreSize = numOfCpus * 2
```

Comment:
Btw, I think that this is a bit questionable, but that is for a different debate.

I think you've already seen this before :smile_cat: 
https://twitter.com/impurepics/status/987758585722621957?lang=en

Reply:
The theory is that num of CPUs is the right number but in practice (and I've seen this discussed somewhere long in the past) is that a bit higher number might work better. And I think I also got inspired in ZIO runtime. Anyway, I don't think there's much difference so if you show me at least some examples or documents (not just one image ;) ) I think we could change it.